### ISSUE TITLE
Cannot start from scratch "XLIFF" translation

### ISSUE THREAD

**Issue by antonkomarev** [WeblateOrg/weblate#8044]
### Describe the issue

I'm trying to create a new component from the scratch, but "File format" select doesn't have XLIFF format.

### I already tried

- [X] I've read and searched [the documentation](https://docs.weblate.org/).
- [X] I've searched for similar issues in this repository.

### Steps to reproduce the behavior

1. Go to "Add new translation component" https://weblate.example/create/component/?project=1#scratch
2. Click on "Start from scratch" tab
3. Click on "File format" select menu

### Expected behavior

XLIFF 1.2 file format exists in "File format" select menu

### Screenshots

<img width="1440" alt="Screenshot 2022-08-23 at 14 18 12" src="https://user-images.githubusercontent.com/1849174/186145031-2a49ac06-1e47-497c-aaac-a8a8a413e4ae.png">

### How do you run Weblate?

Docker container

### Weblate versions

Weblate 4.12.2

**Comment by nijel** [comment]
Weblate currently doesn't have a template for a blank file here, it needs to be added.

**Comment by dankrzeminski32** [comment]
Hi @nijel, Can I work on this?

**Comment by nijel** [comment]
@dankrzeminski32 Yes. I apologize for late reply, I somehow missed this issue. Generally, it's okay just to start working on an issue while writing it there – just to avoid somebody else spending time on it as well.

**Comment by ivan-avalos** [comment]
Any update on this?

**Comment by nijel** [comment]
None so far but I can implement it today...
